source/_patterns/utility-classes/_alignment.scss

```diff
@@ -2,20 +2,31 @@
 // Alignment utility classes
 
 // Align items
-.u-align-left {
+.u-align-left,
+.align-left {
   @include breakpoint(gesso-breakpoint(tablet)) {
     float: left;
     margin-right: rem(gesso-get-map(gutter-width));
   }
 }
 
-.u-align-right {
+.u-align-right,
+.align-right {
   @include breakpoint(gesso-breakpoint(tablet)) {
     float: right;
     margin-left: rem(gesso-get-map(gutter-width));
   }
 }
 
+.u-align-center,
+.align-center {
+  margin-bottom: rem(gesso-get-map(gutter-width));
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: rem(gesso-get-map(gutter-width));
+  text-align: center;
+}
```

Comment:
Why are top & bottom margins added here?

I don’t think `text-align: center` should be here. I often see centered images with caption, but the caption is still left-aligned.

Reply:
You're right -- I've removed those.